Dockerfile.openssl

```diff
@@ -58,15 +58,27 @@ RUN sed -i -e '/OPENSSL_USE_STATIC_LIBS/d' CMakeLists.txt && \
     install -m 755 bin/vfy_ifma_cp_rsa_mb /usr/bin/ && \
     install -m 755 bin/vfy_ifma_rsa_mb /usr/bin/
 
+WORKDIR /intel-ipsec-mb
+RUN make && make install LIB_INSTALL_DIR=/usr/lib64
+
 WORKDIR /QAT_Engine
 RUN ./autogen.sh && \
     ./configure \
-    $QAT_ENGINE_FLAGS \
+    --enable-ipsec_offload \
+    --enable-multibuff_offload \
+    --with-openssl_install_dir=/usr/lib64 \
+    --with-multibuff_install_dir=/usr/local \
+    --enable-openssl_install_build_arch_path && \
+    sed -i -e s:\"qat\":\"avx512\": e_qat.c && \
```

Comment:
Do I understand it right that you are building the engine two times (`avx512.so` and `qat.so`)? I'm just thinking that this `sed` call looks bit fragile for future QAT engine releases. Is there no way to configure the engine run-time to use a particular acceleration method?

Reply:
Currently there's no way to select the acceleration method runtime for rsa mb vs rsa QAT and either one must be select built time. With this PR, I wanted to split the functionality to qat-only and avx512-only and provide users configuration options outside. I can make the `sed` filter more robust...

Reply:
There's `DISABLE_QAT_OFFLOAD` flag for the engine config, but maybe that's indeed not enough.

Reply:
Right. Even if you specify "enable QAT" and "enable RSA MB" config options build-time, only the former is enabled if QAT lib is found from the build env: https://github.com/intel/QAT_Engine/blob/master/.tools/configure.ac#L244-L255

Reply:
Alright. I filed an issue to QAT engine (https://github.com/intel/QAT_Engine/issues/158).

Reply:
I updated the `sed` magic.